core/modules/replica/ReplicateJob.cc

```diff
@@ -1,4 +1,4 @@
-/*
+    /*
  * LSST Data Management System
  * Copyright 2017 LSST Corporation.
```

Comment:
2017-2019?

Reply:
Formally, yes, you're right! In reality we're long overdue to replace these repeated copyrights in all headers an CC files with one file per folder `Copyright` where all that legal stuff would be placed. This would reduce the amount of noise in the files. Someone should probably swallow the pain and migrate our code to that.

Reply:
Having said that, I would probably leave this as is for now and wait before the "Copyright migration"